### DISCUSSION TITLE
Display the type of cellular network (5G, 4G...)

### DISCUSSION THREAD

**Discussion by pchampin** [andyholmes/gnome-shell-extension-gsconnect#1752] [Ideas]
### Describe your request

I would like the popup window of GSConnect to display the type of cellular network (5G, 4G, ...) currently in use by my phone.

### Proposed solution

The mention "5G", "4G", "3G" could be displayed next to the "triangle" indicating the strength of the connection.

### Alternatives

I understand that this information may not be reported by the KDE connect Android application, in which case this feature can not be implemented by GSConnect.

### GSConnect version

55

### Installed from

Extension Manager

### GNOME Shell version

44.3

### Linux distribution/release

Ubuntu 23.04

**Comment by alsoGAMER** [comment]
It already does that, only 5g doesn't have a proper icon yet, so it defaults to the generic cell tower one 
![](https://files.catbox.moe/onc3p4.png)

**Comment by pchampin** [comment]
Oh, I didn't realize that the cell-tower icon carried that info !...

It is not really self-explanatory, though, and on my machine, the tooltip on the cell-tower icon is empty...

**Comment by alsoGAMER** [comment]
It's empty probably because it doesn't know what 5G is, try again forcing the phone to use 2/3/4G

**Comment by pchampin** [comment]
> It's empty probably because it doesn't know what 5G is, try again forcing the phone to use 2/3/4G

As a matter of fact, I have a 4G phone, so the cause if the issue is not the lack of support for 5G.

**Comment by andyholmes** [comment]
Moving this to discussions, since it's not clear this is actually a bug.
